packages/react/src/helpers/validatePassword.ts

```diff
@@ -0,0 +1,72 @@
+const PASSWORD_VALIDATIONS: Array<{ validation: RegExp; errorCode: number }> = [
+  {
+    // Password length validation
+    validation: /^[a-zA-Z0-9\W]{10,}$/,
+    errorCode: 1201000,
+  },
+  {
+    // Password lower case validation
+    validation: /(?=.*[a-z])/,
+    errorCode: 1201001,
+  },
+  {
+    // Password upper case validation
+    validation: /(?=.*[A-Z])/,
+    errorCode: 1201002,
+  },
+  {
+    // Password special character validation
+    validation: /(?=.*\W)/,
+    errorCode: 1201003,
+  },
+  {
+    // Password numbers validation
+    validation: /(?=.*\d)/,
+    errorCode: 1201004,
+  },
+];
+
+/**
+ * @typedef {object} ValidatePasswordResult
```

Comment:
No need for these typedefs. You can create a typescript type `ValidatePasswordResult` and place the documentation there.

Reply:
Reviewed the types, please check the file once again